Firestore/Source/Public/FIRFirestoreSettings.h

```diff
@@ -18,6 +18,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** Used to set on-disk cache size to unlimited. Garbage collection will not run. */
+extern const long kFIRFirestorePersistenceCacheSizeUnlimited;
```

Comment:
Didn't we agree to make this CacheSizeUnlimited?

Also needs an `NS_SWIFT_NAME` and a swift-language use test in `BasicCompileTests.swift` to ensure that the name is mapped correctly out to swift users.